generators/app/templates/.huskyrc

```diff
@@ -0,0 +1,6 @@
+{
+  "hooks": {
+    "pre-commit": "yarn build",
```

Comment:
after adding back tslint, pre-commit hook should run yarn lint instead of yarn build